README.md

```diff
@@ -63,6 +63,9 @@ run OpenHands in a scriptable [headless mode](https://docs.all-hands.dev/modules
 interact with it via a [friendly CLI](https://docs.all-hands.dev/modules/usage/how-to/cli-mode),
 or run it on tagged issues with [a github action](https://github.com/All-Hands-AI/OpenHands/blob/main/openhands/resolver/README.md).
 
+Follow the [Persisting Session Data](https://docs.all-hands.dev/modules/usage/how-to/persist-session-data) guide if you
+would like your session data to persist.
```

Comment:
@enyst and @rbren I am confused about something. If this is something that's important enough to mention in the README, does this mean this is a bug that we currently don't persist the session data?
Also if it's this important, why would we not add this to the docker command above?

Reply:
IMO it's not super important right now. Most users probably start the app, do stuff with it, then shut down--no need to persist a single session across upgrades.

That said, in multi-conversation this will become more important, and we'll probably want to add it to the default setup, despite the fact that that increases complexity a bit

Reply:
I don't know how many users are using it seriously. But the problem with the current setup is that as long as it stays, you kinda have to use for toy projects, or know what you're doing, no much of a middle ground.

IMO we should add it to the docker run command.

Reply:
This is tough. Do you think this PR is mergeable as is for now? It moves the documentation to another part and makes it a little better? :D 

Not sure if we are going to get this added to the main docker command.

Reply:
Sorry, sure!